### ISSUE TITLE
Command line argument to get html of a page using page_id

### ISSUE THREAD

**Issue by shopuz** [gnosygnu/xowa#134]
Right now, XOWA has this app.shell.fetch_page which accepts the page_name.
`java -jar xowa_windows_64.jar --app_mode cmd --show_license n --show_args n --cmd_text "app.shell.fetch_page('en.wikipedia.org/wiki/Earth' 'html');"
`

**Is it possible to have similar feature with page id ?**
Say:
`java -jar xowa_windows_64.jar --app_mode cmd --show_license n --show_args n --cmd_text "app.shell.fetch_page('en.wikipedia.org/wiki/9228' 'html');"`
where **9228** is the **page_id** for the article **Earth**..

**Comment by gnosygnu** [comment]
Oops. It looks like tonight's release doesn't support ?curid=9228 in command-line. Sorry about that.

I think this is an easy fix on my side. I can upload a new version tomorrow for you if you want. Let me know which OS and bitness (EX: Linux 64-bit) and I'll make a build.

Thanks.

**Comment by shopuz** [comment]
Linux 64-bit should be fine @gnosygnu 
Thank you for the quick release 👍

**Comment by gnosygnu** [comment]
No problem. Just generated it now and tested on my windows box. I ran it with the cmd below:

```
java -jar xowa_windows_64.jar  --app_mode cmd --show_license n --show_args n --cmd_text "app.shell.fetch_page('en.wikipedia.org/wiki/?curid=9228' 'html');"
```

Here's the link: https://drive.google.com/open?id=0B9cb52zjL2rINl9mSUhQMWp6UXc

Let me know how it goes. Thanks!

**Comment by shopuz** [comment]
Tested and it works perfect !!!

Thanks @gnosygnu  👍

**Comment by gnosygnu** [comment]
@shopuz Cool. Glad to hear!

By the way, I like to credit others for reporting defects. There will be a line in the Change Log with something like "Support ?curid for command-line calls {detected by anonymous}". Let me know if you want if you would like to be cited by nickname or rather remain anonymous.

Otherwise, I'll close this ticket out after the next official release.

Thanks!

**Comment by shopuz** [comment]
I would prefer `shopuz` nickname @gnosygnu 
Thanks a lot again :)

**Comment by gnosygnu** [comment]
@shopuz Cool. <code>shopuz</code> it is.

Thanks!

**Comment by gnosygnu** [comment]
As this fix was released in tonight's v4.4.0 release, I'm going to close this issue out. Thanks again for the report!
